packages/dds/shared-object-base/src/summarySerializer.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { FluidSerializer } from "@fluidframework/runtime-utils";
+
+/**
+ * Serializer implementation for serializing handles during summary.
+ */
+export class SummarySerializer extends FluidSerializer {
+    private readonly _serializedRoutes: Set<string> = new Set();
+    public get serializedRoutes(): string[] {
```

Comment:
nit. i genreally don't like to hide costly operations (array copy and allocate) like  this under properties as the consumer may call it repeatedly. putting it under a method encourages the consumer to cache the result

Reply:
That is a great point! I will change it and keep this in mind for future changes, thanks! :-)